src/svxlink/svxlink/locale.tcl

```diff
@@ -10,18 +10,23 @@
 # Spell the specified word using a phonetic alphabet
 #
 proc spellWord {word} {
+  global phonetic_spelling
```

Comment:
You can implement this without modifying the C++ code since any configuration variable is available in the TCL code as CFG_varname. So if you specify PHONETIC_SPELLING in the config you can access it as CFG_PHONETIC_SPELLING in the TCL code. E.g.:
```
variable Logic::CFG_PHONETIC_SPELLING
puts "### PHONETIC_SPELLING=$CFG_PHONETIC_SPELLING"
```